src/wallet/wallet.cpp

```diff
@@ -3305,12 +3312,18 @@ bool CWallet::CreateTransaction(interfaces::Chain::Lock& locked_chain, const std
                             strFailReason = _("Transaction amount too small");
                         return false;
                     }
+
+                    // Include the fee cost for outputs. Note this is only used for BnB right now
+                    coin_selection_params.tx_noinputs_size += ::GetSerializeSize(txout, PROTOCOL_VERSION);
```

Comment:
I'm confused that we weren't already doing this? Isn't this a significant fraction of the size?

Reply:
Oh it just moved, I should read more carefully. Nevermind.